metadata.yaml

```diff
@@ -3,13 +3,4 @@ description: |
   Placeholder charm
 summary: |
   Placeholder charm
-website: https://github.com/canonical/ip-router-interface
```

Comment:
This change is not related to the PR title or description, please create a separate PR for it.

Reply:
removed